docs/operation/cluster_and_service_configuration.rst

```diff
@@ -30,12 +32,11 @@ MetalK8s installation.
 
 To add a new static user, perform the following operations:
 
-#. Generate a password ``hash``. The command below requires **apache-utils**
-   package to be installed.
+#. Generate a bcrypt hash of your new password.
 
    .. code-block:: shell
 
-      root@bootstrap $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+      root@bootstrap $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':'
```

Comment:
I think the point of this ticket was to not rely on some external package, but build this hash with embedded dependencies...

I'd rather ship a simple Salt module to generate it from the available tooling and document this (in the future, we'll be able to reuse such a module in our CLI).

Reply:
In regards to this, out of the box, Salt alone will not work here since the hash is a  `bcrypt` with work factor of atleast 12. Salt does not come with the bcrypt package as checked meaning we need to install this external package.

For a start, we could package bcrypt in the ISO, but we already have httpd-tools added to metalk8s-utils image as from this PR yesterday: https://github.com/scality/metalk8s/pull/2374 which seems sufficient for me given that we have docs on how to run the container

Btw, this ticket is really about updating the documentation to include default login credentials

Reply:
Oh, right. You're changing the lines on how to provision users, so I thought you were also tackling #2352. If this PR really is only about default creds, then there's a lot of changes that are not related and would need another PR, don't you think?

Reply:
If we want to use the `metalk8s-utils` container image for generating this, we could just provide the full command to run. Maybe (not tested yet) something like:

```shellscript
$ kubectl --kubeconfig=/etc/kubernetes/admin.conf \
    run -it tmp-gen-pwd \
    --image=metalk8s-registry-from-config.invalid/metalk8s-|release|/metalk8s-utils:|release| \
    --command -- htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':'
```

Reply:
I probably will rewrite this PR to include your comments here: https://github.com/scality/metalk8s/pull/2377#discussion_r404694946 then close the two tickets